Source/JavaScriptCore/assembler/X86Assembler.h

```diff
@@ -3682,6 +3699,111 @@ class X86Assembler {
+    {
+        m_formatter.vexNdsLigWigTwoByteOp((OneByteOpcodeID)0, OP2_ANDNPS_VpsWps, (RegisterID)dest, (RegisterID)a, (RegisterID)b);
+    }
+
+    void vandnpd_rrr(XMMRegisterID a, XMMRegisterID b, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_ANDNPD_VpdWpd, (RegisterID)dest, (RegisterID)a, (RegisterID)b);
+    }
+
+    void vpmovmskb_rr(XMMRegisterID input, RegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PMOVMSKB_EqWdq, (RegisterID)dest, (RegisterID)0, (RegisterID)input);
+    }
+
+    void vmovmskps_rr(XMMRegisterID input, RegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp((OneByteOpcodeID)0, OP2_MOVMSKPS_EqWps, (RegisterID)dest, (RegisterID)0, (RegisterID)input);
+    }
+
+    void vmovmskpd_rr(XMMRegisterID input, RegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_MOVMSKPD_EqWpd, (RegisterID)dest, (RegisterID)0, (RegisterID)input);
+    }
+
+    void vptest_rr(XMMRegisterID a, XMMRegisterID b)
+    {
+        m_formatter.vexNdsLigWigThreeByteOp(PRE_OPERAND_SIZE, VexImpliedBytes::ThreeBytesOp38, OP3_PTEST_VdqWdq, (RegisterID)a, (RegisterID)0, (RegisterID)b);
+    }
+
+    void vpsllw_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSLLW_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpslld_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSLLD_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsllq_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSLLQ_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsrlw_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSRLW_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsrld_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSRLD_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsrlq_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSRLQ_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsraw_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
+    {
+        m_formatter.vexNdsLigWigTwoByteOp(PRE_OPERAND_SIZE, OP2_PSRAW_VdqWdq, (RegisterID)dest, (RegisterID)input, (RegisterID)shift);
+    }
+
+    void vpsrad_rrr(XMMRegisterID input, XMMRegisterID shift, XMMRegisterID dest)
```

Comment:
I think these ordering needs to be changed to align to ATT style. Currently, it is not aligned.